Sources/AblyChat/RoomLifecycleManager.swift

```diff
@@ -681,7 +710,7 @@ internal actor DefaultRoomLifecycleManager<Contributor: RoomLifecycleContributor
 
         // CHA-RL1d
         if let currentOperationID = status.operationID {
-            try? await waitForCompletionOfOperationWithID(currentOperationID, waitingOperationID: operationID)
+            try? await waitForCompletionOfOperationWithID(currentOperationID, requester: .anotherOperation(operationID: operationID))
```

Comment:
If `waitForCompletionOfOperationWithID` throws nothing will happen?

Reply:
Correct — CHA-RL1d just says "** @(CHA-RL1d)@ @[Testable]@ If a room lifecycle operation is already in progress, this operation shall wait until the current operation completes before continuing"; it doesn't say to use the result of that operation.